ui/src/containers/EquipmentContainer.jsx

```diff
@@ -46,142 +46,136 @@ import HarvesterMaintenance from '../components/Equipment/HarvesterMaintenance';
 import GenericEquipment from '../components/Equipment/GenericEquipment';
 import GenericEquipmentControl from '../components/Equipment/GenericEquipmentControl';
 
-class EquipmentContainer extends React.Component {
-  render() {
-    return (
-      <Container fluid className="mt-3">
-        <Row className="d-flex">
-          <Col sm={12}>
+function EquipmentContainer(props) {
+  return (
+    <Container fluid className="mt-3">
+      <Row className="d-flex">
+        <Col sm={12}>
+          <GenericEquipment
+            state={props.sampleChangerState}
+            name={props.contents?.name}
+            initialOpen
+          >
+            {props.contents.name === 'PlateManipulator' ? (
+              <Row className="row">
+                <Col sm={6}>
+                  <PlateManipulator
+                    contents={props.contents}
```

Comment:
Eventually, all these props coming from the Redux state should go away. Each component should retrieve its own state information with `useSelector`.